build/manager/Dockerfile

```diff
@@ -24,6 +24,9 @@ RUN swag init -g handlers/doc.go
 # Build the Go application
 RUN go build -o bin/manager cmd/manager/main.go
 
+# Build the migration tool
+RUN go build -o bin/migrate cmd/tooling/main.go
```

Comment:
Tooling represent a general cli, which might have other methods in the future. I would consider naming it as such.